### PR TITLE
Update createmail.js

### PR DESCRIPTION
**Pull request by yashshah2820**
Fixed typo error (from you email to your email) in line 134

### CODE REVIEW

File: createmail.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ const createMail = function(username, email) {
             <main>
                 <h3>We're glad to have you with us, ${username}!</h3>
                 <br>
-                <p>Click on the button below to verify you e-mail ID,<br>and you are good to go!</p>
+                <p>Click on the button below to verify your e-mail ID,<br>and you are good to go!</p>
                 <a href='${email}' id='verify'>Verify e-mail</a>
```
Comment (aashutoshrathi) [inline createmail.js]: Make it `Verify Identity`

**Comment by yashLadha** [comment]
Status @yashshah2820

**Comment by yashshah2820** [comment]
Sir, I have committed some changes in the e-mail's content in new PR.

I am closing this PR.

**Comment by aashutoshrathi** [comment]
Sure

**Pull request closed without merging**